Problem adding this library as a dependency

There is an issue with including this library by itself where classes are not able to be found at runtime. Stacktrace:

```
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.google.maps.android.SphericalUtil" on path: DexPathList[[zip file "/data/app/com.google.codelabs.findlocalplacesinar-IlY-Zx_cOMFwQ5RPtmGgug==/base.apk"],nativeLibraryDirectories=[/data/app/com.google.codelabs.findlocalplacesinar-IlY-Zx_cOMFwQ5RPtmGgug==/lib/x86, /data/app/com.google.codelabs.findlocalplacesinar-IlY-Zx_cOMFwQ5RPtmGgug==/base.apk!/lib/x86, /system/lib, /system/product/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:196)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
```


My `build.gradle` looks like this:

```
dependencies {
    implementation 'com.google.maps.android:maps-utils-ktx:0.1.1'
}
```

PR #10 was meant to solve this but looks like there are still issues.

--- a/build.gradle
+++ b/build.gradle
@@ -140,6 +140,16 @@ subprojects { project ->
                     artifact dokkaJar
                     artifact sourcesJar
                 }
+
+                pom.withXml {
+                    def dependenciesNode = asNode().appendNode('dependencies')
+                    project.configurations.api.allDependencies.each { dependency ->
+                        def dependencyNode = dependenciesNode.appendNode('dependency')
+                        dependencyNode.appendNode('groupId', dependency.group)
+                        dependencyNode.appendNode('artifactId', dependency.name)
+                        dependencyNode.appendNode('version', dependency.version)
+                    }
+                }
             }
         }
 

--- a/maps-utils-ktx/build.gradle
+++ b/maps-utils-ktx/build.gradle
@@ -40,6 +40,9 @@ android {
         }
     }
 
+    libraryVariants.all {
+        it.generateBuildConfig.enabled = false
+    }
 }
 
 dependencies {

